R/extract.R

```diff
@@ -5147,6 +5147,47 @@ extract.pglm <- function(model, include.aic = TRUE,
     )
     return(tr)
 }
-
 setMethod("extract", signature = className("pglm", "pglm"),
           definition = extract.pglm)
+					 
+					 
+#extention for panelAR objects (panelAR package)					 
+extract.panelAR <- function(model, include.rsquared=TRUE, include.nobs=TRUE, include.groups=TRUE,...){
+    s <- summary(model,...)
+    names <- rownames(s$coef)

```

Comment:
can you please replace "names" with "coefficient.names"? --names is an inbuilt R function and might generate conflicts.